Assets/GoogleVR/Plugins/iOS/GvrDaydreamController.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import "daydream_controller_state.h"
+
+@interface GvrDaydreamController : NSObject
+
+- (daydream_controller_state)getState;
+
+- (void)pause;
+
+- (void)resume;
+
+- (void)start;
+
++ (GvrDaydreamController *)sharedInstance;
```

Comment:
This singleton instance is used, so it currently supports connecting to a single controller at a time.